exchanges/README.md

```diff
@@ -22,6 +22,9 @@ Join our slack to discuss all things related to GoCryptoTrader! [GoCryptoTrader
 
 + This package is used to connect and query data from supported exchanges.
 
++ A guide on implementing API for a new exchange can be found [here.]
```

Comment:
A guide on how to add an exchange to the GoCryptoTrader framework can be found [here](../docs/ADD_NEW_EXCHANGE.md).

On another note, I think this should have its own heading and moved out of current features